### ISSUE TITLE
Fire Dragon Nesting Site

### ISSUE THREAD

**Issue by Aralabie** [AlexModGuy/Ice-and-Fire-Suggestions#1168]
I have a suggestion that Fire dragons should nest in the nether personaly I think it would make more sense

**Comment by CTH999** [comment]
I disagree, but I think there should be smaller, faster "lava dragons" in the nether. They are smaller, faster, breath plumes of magma, shoot out molten boulders, and can "phase" through nether-rack (If they touch netherak, they can go in a straight line and pop out in the nearest air pocket). They are always smaller, so only the full-grown ones can be ridden, and are about the size of a horse. But, they can nearly outmatch a lvl. 5 dragon, if ridden correctly! They grow in magma, and are highly intelligent, attacking in swarms. MWAHAHAHAHA!!!
